includes/Core/REST_API/Data_Request.php

```diff
@@ -121,7 +121,9 @@ public function __isset( $name ) {
 	 *
 	 * @return bool
 	 */
-	public function offsetExists( $key ) {
+	// phpcs:ignore Squiz.Commenting.InlineComment.WrongStyle,Squiz.PHP.CommentedOutCode.Found
+	#[\ReturnTypeWillChange]
+	public function offsetExists( $key ) { // phpcs:ignore Squiz.Commenting.FunctionComment.WrongStyle
```

Comment:
When I run `phpcs` with PHP 8, these lines still raise errors:

![image](https://user-images.githubusercontent.com/18395600/207650523-71340bb9-8f66-4fc1-9536-07332c4ebdb3.png)

Loosening up the rule definition allows linting to pass in PHP 8 as well as older versions.

```suggestion
	public function offsetExists( $key ) { // phpcs:ignore Squiz.Commenting.FunctionComment
```